style.css

```diff
@@ -0,0 +1,1043 @@
+  width: 5rem;
+  height: 4.5rem;
+  margin-right: 0.2rem;
+}
+
+.ac2-secondary-article * {
+  color: var(--color-orange);
+  display: block;
+  width: 95%;
+  margin: 0.4rem auto;
+}
+
+.ac2-secondary-article h5 {
+  font-size: 0.9rem;
+}
+
+.article-content-2 ul {
+  margin-top: 0.5rem;
+  margin-left: 0.75rem;
+}
+
+footer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin-top: 5rem;
+  width: 100%;
+}
+
+.footer-section-1,
+.footer-section-2 {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  justify-content: center;
+  align-items: center;
+}
+
+.footer-section-1 {
+  background-color: rgb(19, 19, 19, 0.98);
+  height: 10rem;
+}
+
+.footer-navbar {
+  width: 20rem;
+}
+
+footer ul:first-child {
+  margin-bottom: 2rem;
+}
+
+.footer-navbar * {
+  font-family: var(--chakra-font);
+  font-size: 0.8rem;
+  opacity: 0.8;
+}
+
+.footer-section-2 {
+  display: flex;
+  flex-direction: row;
+  color: var(--color-white);
+  font-size: 0.65rem;
+  background-color: var(--color-black);
+  height: 6rem;
+}
+
+.footer-logo {
+  width: 2rem;
+  height: 0.5rem;
+  margin: 0 0.3rem;
+}
+
+.heart-logo {
+  width: 0.4rem;
+  height: 0.4rem;
+  margin: 0 0.3rem;
+}
+
+@media all and (max-width: 1114px) {
+  header p {
+    text-align: center;
+  }
+
+  .main-header-content {
+    display: grid;
+    grid-template-columns: 1fr 13fr;
+  }
+
+  .gradient-logo img {
+    display: block;
+  }
+
+  .nav-2-ul-2-text {
+    display: none;
+  }
+
+  .countdown {
+    display: none;
+  }
+
+  .nav-2-ul-2 button {
+    display: none;
+  }
+}
+
+@media all and (max-width: 1024px) {
+  header p {
+    padding-top: 1rem;
+    padding-bottom: 1rem;
+  }
+
+  .button-li {
+    display: none;
+  }
+
+  .hero-article div:first-child {
+    visibility: hidden;
+  }
+
+  .hero-article {
+    background: linear-gradient(to bottom right, rgba(0, 0, 0, 0.1), rgba(0, 0, 0, 0.9)), url("https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/03/bmw-car-key-apple-carkey-digital-key-car-unloack-phone-ios-code-796x448.jpg") center center scroll no-repeat;
+    background-size: cover;
+  }
+
+  .hero-article h1 {
+    font-size: 2rem;
+  }
+
+  .main-article-aside h1,
+  .main-article-aside h2 {
+    font-size: 1rem;
+  }
+
+  .grid-header {
+    width: 90%;
+  }
+
+  .latest-news-grid {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    width: 90%;
+  }
+
+  .latest-news-grid img {
+    height: 11rem;
+  }
+
+  .latest-deals-grid div img {
+    height: 11rem;
+  }
+
+  .mid-article-head-container {
+    width: 90%;
+    display: grid;
+    grid-template: 4rem 3fr / 1fr;
+    margin: 4rem auto;
+  }
+
+  .mid-article-head-title {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+  }
+
+  .mid-article-head-title ul {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    height: 1rem;
+  }
+
+  .last-child {
+    display: none;
+  }
```

Comment:
This selector and selector in line 774 have the same style definition. Group selectors to minimize code.

Reply:
Implemented